pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -1278,10 +1281,16 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject) in
 	go objectInfoTask(&zedagentCtx, triggerObjectInfo)
 	log.Functionf("Creating %s at %s", "flowLogTask", agentlog.GetMyStack())
 	go flowlogTask(&zedagentCtx, flowlogQueue)
+	log.Functionf("Creating %s at %s", "hardwareInfoTask", agentlog.GetMyStack())
+	go hardwareInfoTask(&zedagentCtx, triggerHwInfo)
 
 	// Publish initial device info.
 	triggerPublishDevInfo(&zedagentCtx)
 
+	// Publish initial hardware info.
+	hwInfoTiker := time.NewTicker(3 * time.Hour)
```

Comment:
3 hours would be rare enough?

Reply:
Do we also trigger this information if there is some important change?

Reply:
Three hours is reasonable for now. Down the road we can make it configurable using a configItem

Reply:
> Do we also trigger this information if there is some important change?

We can run this, for example, after a disk swap operation (when it is will be implemented), or in other places as the HardwareInfo message grows